lazy_tensor_core/lazy_tensor_core/csrc/tensor_ops.cpp

```diff
@@ -67,17 +67,6 @@ LazyTensor Cross(const LazyTensor& input, const LazyTensor& other,
   return lazy_tensor_aten_ops::stack({s1, s2, s3}, canonical_dim);
 }
 
-LazyTensor MakeMatrixWithDiagonal(const LazyTensor& input, int64_t diagonal) {
```

Comment:
I see this is used by the deleted `diag` function, but I expect `diag` still needs to be supported. Is the intention of this diff to remove functions for all autogen-ed ops? If yes, maybe we should focus on ops in the `full_codegen` section.

Reply:
I asked @alanwaketan to help clean up unused ops code, meaning code for ops that we don't yet support in the TS backend.  My rationale was that each time we do a refactor to clean up other interfaces/utils, these extra ops slow us down, and our aim is to move (most of) them to codegen anyway.  I think if we find that we need some helper function we deleted, it won't be hard to get it back from our history or pt/xla codebase; but if you prefer keeping some of it I'm fine with that too.

Reply:
All the deleted ops are not in the supported list (ts_native_functions.yaml), and not being code-generated as well.

Reply:
Sounds ok to me. History is supposed to be tracked by git, and we can always bring things back later if needed.